test/ecto/adapters/postgres_test.exs

```diff
@@ -116,6 +116,15 @@ defmodule Ecto.Adapters.PostgresTest do
     assert all(query) == ~s{SELECT count(DISTINCT s0."x") FROM "schema" AS s0}
   end
 
+  @tag :aggregate_filters
```

Comment:
We don't need to tag here because this is a unit test. It will always work regardless of version as the database is not involved.

Reply:
And we need a similar test for MySQL, except it will raise. :)